Dockerfile

```diff
@@ -0,0 +1,30 @@
+FROM php:apache
+MAINTAINER David A. Lareo "dalareo@gmail.com"
+
+RUN apt-get update -y \
+	&& apt-get install -y git libmcrypt-dev libldap2-dev \
+	&& docker-php-ext-configure ldap --with-libdir=lib/x86_64-linux-gnu/ \
+
+	&& docker-php-ext-install mcrypt ldap
+
+RUN git clone https://github.com/simplesamlphp/simplesamlphp.git /var/simplesamlphp
```

Comment:
Yes, you're write that first cloning was a mistake. It makes sense to clone the repo instead of just copying the code in order to let the file be able to work aside the original repo.